### PR TITLE
Show # genes matched as integer in PatientView

### PR DESCRIPTION
**Pull request by msalihaltun**
Address point 3 in https://github.com/cBioPortal/cbioportal/issues/8078#issuecomment-732722936.

Change `package.json` to point to latest commit in PathwayMapper where point 1 in above comment is addressed.

### CODE REVIEW

File: src/pages/patientView/pathwayMapper/PatientViewPathwayMapper.tsx [hunk 1]
```diff
@@ -170,6 +170,7 @@ export default class PatientViewPathwayMapper extends React.Component<
                         tooltip: <span>Number of Genes Matched</span>,
                     },
```
Comment (onursumer) [inline src/pages/patientView/pathwayMapper/PatientViewPathwayMapper.tsx:176]: Let's just define a custom render function here for the score column. This way, you wouldn't need to modify the default render function inside `PathwayMapperTable`, and you wouldn't need to define another prop like `convertScoreRepresentation`.

```suggestion
                        render: (d: IPathwayMapperTable) => (
                            <span>
                                <b>{d.score.toFixed(0)}</b>
                           </span>
                       ),
                    },
```

File: src/shared/lib/pathwayMapper/PathwayMapperTable.tsx [hunk 2]
```diff
@@ -133,11 +134,18 @@ export default class PathwayMapperTable extends React.Component<
 
         this._columns[IPathwayMapperTableColumnType.SCORE] = {
             name: 'Score',
-            render: (d: IPathwayMapperTable) => (
-                <span>
-                    <b>{d.score.toFixed(2)}</b>
-                </span>
-            ),
+            render: (d: IPathwayMapperTable) => {
```
Comment (onursumer) [inline src/shared/lib/pathwayMapper/PathwayMapperTable.tsx]: Let's just leave the default render as is. See my comments about an alternative solution.

File: src/shared/lib/pathwayMapper/PathwayMapperTable.tsx [hunk 3]
```diff
@@ -34,6 +34,7 @@ interface IPathwayMapperTableProps {
     columnsOverride?: {
         [columnEnum: number]: Partial<PathwayMapperTableColumn>;
     };
+    convertScoreRepresentation?: (score: number) => string;
```
Comment (onursumer) [inline src/shared/lib/pathwayMapper/PathwayMapperTable.tsx]: Let's remove this prop.

**Review by onursumer — CHANGES_REQUESTED** [review]
Let's just simplify this PR by defining a custom render function for the score column.

File: src/pages/patientView/pathwayMapper/PatientViewPathwayMapper.tsx (hunk 1 above)
Comment (msalihaltun) [reply src/pages/patientView/pathwayMapper/PatientViewPathwayMapper.tsx:176]: I didn't know I could do that :)

Comment (onursumer) [reply src/pages/patientView/pathwayMapper/PatientViewPathwayMapper.tsx:176]: no worries :)

**Review by onursumer — APPROVED** [review]

**Pull request merged**